templates/travel/partials/util/apply-button-props.snip.html

```diff
@@ -0,0 +1,35 @@
+{{!
+<!---
+Copyright 2016 The AMP Start Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+on="
```

Comment:
this clearly does not warrant a partial , move all the attributes to JSON and render this in mustache

Reply:
Looked into this more. I see why standards-wise it would be best to use amp-state to bind elements to a JSON state. However, because the central expression that we use to pass state around the app is at already at the max complexity limit, we can’t store state in nested objects. Would it be better if these attributes were declared in the element?

Reply:
all i am implying here is: 

1. A partial is a snippet that would be reused 
2. It has to be a functional piece of code (which this is not)

So it is best you can use the JSON files and use mustache loops to render this in HTML. Do not create a new .snip file for this.

Reply:
Like this? https://github.com/ampproject/ampstart/pull/676/commits/5580796321879fd28d64ed121d56b8a26a94f231